src/Flex.php

```diff
@@ -39,13 +37,41 @@
  */
 class Flex implements PluginInterface, EventSubscriberInterface
 {
+    /**
+     * @var Composer
+     */
     private $composer;
+
+    /**
+     * @var IOInterface
+     */
     private $io;
+
+    /**
+     * @var Options
+     */
     private $options;
+
+    /**
+     * @var Configurator
+     */
     private $configurator;
+
+    /**
+     * @var Downloader
+     */
     private $downloader;
+
+    /**
+     * @var array
+     */
     private $postInstallOutput = [''];
+
+    /**
+     * @var \Closure
+     */
```

Comment:
I'm not using an IDE, so there are useless for me :) Just wanted to get some more feedback on that.